source/wp-content/themes/wporg-parent-2021/theme.json

```diff
@@ -121,7 +121,9 @@
 		},
 		"custom": {
 			"alignment": {
-				"alignedMaxWidth": "50%"
+				"alignedMaxWidth": "50%",
+				"//edge-spacing": "The min and max values represent the mobile and desktop spacing; the calc() value is for tablets.",
+				"edge-spacing": "clamp( 24px, calc( 100vw / 18 ), 80px )"
```

Comment:
Rather than adding this value, can you use `var(--wp--preset--spacing--60)`? I've been trying to get us to stick to [the defined spacing values](https://github.com/WordPress/wporg-parent-2021/#spacing), and that one is `clamp( 20px, calc( 100vw / 18 ), 80px )`.

Even the global header (which uses that value) will be changing soon (https://github.com/WordPress/wporg-mu-plugins/issues/301).